tests/testthat/test-load-runner.R

```diff
@@ -13,6 +13,8 @@ testthat::test_that("import fails if log table misconfigured", {
   invisible(DBI::dbExecute(con,
     "CREATE TABLE table_log (
       name   TEXT,
+      language TEXT,
```

Comment:
this is the sort of thing that a schema mismatch error would be nicer for:

"You are trying to import using dettl version 'x.y.x' but need to update your schema: see ?dettlschema for details"

Reply:
Yeah that sounds much nicer, will add a note to the ticket above